media-libs/libsfml/libsfml-2.5.0.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils versionator
```

Comment:
eapi7-ver instead of versionator

Reply:
Yes, but it will be. At that point, versionator will not be usable on EAPI 7. The point I'm making
is if you use `eapi7-ver.eclass` instead of `versionator.eclass` and `ver_cut 1-2` instead of
`get_version_component_range 1-2`, when this package does get bumped to EAPI 7, the only
change needed will be removing `eapi7-ver` from the inherit line.